confuse.py

```diff
@@ -1600,6 +1605,23 @@ def value(self, view, template=None):
         return os.path.abspath(path)
 
 
+class Path(Filename):
+    """A template that validates strings as `pathlib.Path` objects.
+
+    Filenames are parsed equivalent to the `Filename` template and then
+    converted to `pathlib.Path` objects.
+
+    For Python 2 it returns the original path as returned by the `Filename`
+    template.
+    """
+    def value(self, view, template=None):
+        path = super(Path, self).value(view, template)
+        if PY3:
```

Comment:
Perhaps this would be slightly better implemented with `try`/`except ImportError:` handler? That way, we can avoid version numbering shenanigans and even transparently work if a pathlib backport is available on Python 2.

Reply:
Hi, definitely better! I'll do that right away and also remove the Python 3.4 then.

Reply:
I am not very happy with my solution to silently fall back to normal string paths if the import fails. Would it not be better to at least send an UserWarning or raise an error?

Reply:
Hmm; that's a good point… maybe we should just have a plain old `import` here and leave it at that! This will make the failure evident (via an `ImportError`, which seems like the best way to convey the problem) in code that tries to use a library that's not available.

Reply:
Agree; actually that was my original solution, then I wanted to support Python 2 somehow so I made some unnecessary complications without solving the problem... Now it seems right, thanks!